CodeChat_Server/easyinstallnotes.rst

```diff
@@ -0,0 +1,118 @@
+**************************
+Easy Installation and use
+**************************
+
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+
+To install the CodeChat System, install the CodeChat Server and a CodeChat extension or plugin, along with optional external renderers you desire.
+
+.. _install CodeChat Server:
+
+CodeChat Server Installation Notes
+==================================
+
+.. note:: 
+ 
+ the idea behind our entire project is to get this instruction list as short at possible
+
+Windows
+-------
+#.  Go to this `link <https://github.com/JoeKenn1118/CodeChat_system/releases/tag/v0.0.1>`_ and download the Powershell script (``windows_installer.ps1``)
```

Comment:
Update based on Liv and Kaitlyn's scripts -- copy and paste a PowerShell script.